hydrolib/core/dflowfm/crosssection/models.py

```diff
@@ -243,7 +243,7 @@ class Comments(CrossSectionDefinition.Comments):
     type: Literal["rectangle"] = Field("rectangle")
     width: float
     height: float
-    closed: bool = Field(True)
+    closed: str = Field("no")
```

Comment:
Changing the type and inverting the default value is quite breaking. I'm not sure what the question was, but this might be solved by a custom reader/writer (or flag type) that translates the boolean into yes/no strings?